tests/src/dir.targets

```diff
@@ -120,7 +119,8 @@
 
   <Target Name="CopyNativeProjectBinaries">
      <ItemGroup>
-        <NativeProjectBinaries Include="$(NativeProjectOutputFolder)\**\*.*" />
+        <NativeProjectBinaries Include="$(NativeProjectOutputFolder)\**\*.*" Condition="'$(BuildOS)' == 'Windows_NT'"/>
+        <NativeProjectBinaries Include="$(ProjectDir)\..\bin\Product\$(OSPlatformConfig)\bin\*.*" Condition="'$(BuildOS)' != 'Windows_NT'"/>
```

Comment:
@rartemev this copies all the native binaries built to the test directory. while in windows we only copy the ones specified by the associated CMAKE file. 

Please don't copy it here. For the short term you can copy these native binaries into CORE_ROOT, which i assume happens as of today.

For the longer term, we should standardize on the native intermediate folder path and construct one for NativeProjectOutputFolder as that is done in windows today 

makes sense ?